tools/sdk-generation-pipeline/documents/docker/README.md

```diff
@@ -19,11 +19,14 @@ The docker image will be used in different scenarios:
 ### RUN DOCKER CONTAINER TO GENERATE CODES AND DO GROW UP DEVELOPMENT
 
 Command
-
-```shell
-docker run -it --privileged  -v {local_spec_repo_path}:/spec-repo -v {local_work_folder}:/work-dir -v {local_autorest_config}:/autorest.md sdkgeneration.azurecr.io/sdk-generation:latest --readme={relative_readme} --sdk={sdk_to_generate}
-```
-
+1. Generate codes with Autorest:
+    ```shell
+    docker run -it --privileged -v {local_spec_repo_path}:/spec-repo -v {local_work_folder}:/work-dir -v {local_autorest_config}:/autorest.md sdkgeneration.azurecr.io/sdk-generation:latest --readme={relative_readme} --sdk={sdk_to_generate}
+    ```
+2. Generate codes with CADL:
+    ```shell
+    docker run -it --privileged -v {local_spec_repo_path}:/spec-repo -v {local_work_folder}:/work-dir -v sdkgeneration.azurecr.io/sdk-generation:latest --readme={relative_cadl_project} --sdk={sdk_to_generate}
```

Comment:
if should replace --readme with --cadl-project

Reply:
yes. updated.